### ISSUE TITLE
log files are public available

### ISSUE THREAD

**Issue by abma** [springfiles/springrts_logs#11]
they should be protected somehow, not sure what to use for auth...

**Comment by dansan** [comment]
Yeah… until now this service was in testing mode (all open) for lobby developers. If this should go live, auth is a must.

Auth using lobby accounts is no problem (see replay site). I don't know about forum accounts.
The questions are:
* who to allow access?
* allow access to all logs or filter by dev→game?

I suggest to start with "all engine and game devs have access to all uploaded logs".

I can code the authentication/authorization part, but I'm not active enough in the dev community, so I'd like to have someone else managing the account operations.

**Comment by TurBoss** [comment]
I can take care of this

**Comment by dansan** [comment]
Great. Then we'll just need to wait for https://github.com/spring/uberserver/issues/365 and @TurBoss  having write access to the `group_user` table.
The code here will be the same as in the replay site (I guess I should make it a standalone "Django app" with its own git repo) plus: after authenticating the user asking the lobby server if the user is in the "infolog_reader" group (or similar).
